src/Microsoft.Identity.Client/Mats/IActionHandle.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client.Mats
+{
+    internal interface IActionHandle

```

Comment:
A bit general :). Can you add a comment on what this is?

Reply:
This is currently to map to the C API from MATS.  Once we agree we're building this ourselves, I don't think we need this interface type at all.

---
In reply to: [264426428](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/955#discussion_r264426428) [](ancestors = 264426428)